kubernetes-client/src/main/resources/META-INF/services/io.fabric8.kubernetes.client.ResourceHandler

```diff
@@ -23,10 +23,10 @@ io.fabric8.kubernetes.client.handlers.EndpointsHandler
 io.fabric8.kubernetes.client.handlers.EventHandler
 io.fabric8.kubernetes.client.handlers.HorizontalPodAutoscalerHandler
 io.fabric8.kubernetes.client.handlers.IngressHandler
-io.fabric8.kubernetes.client.handlers.KubernetesClusterRoleBindingHandler
-io.fabric8.kubernetes.client.handlers.KubernetesClusterRoleHandler
-io.fabric8.kubernetes.client.handlers.KubernetesRoleBindingHandler
-io.fabric8.kubernetes.client.handlers.KubernetesRoleHandler
+io.fabric8.kubernetes.client.handlers.ClusterRoleBindingHandler
```

Comment:
nitpick, this list seems to be in alphabetical order

Reply:
Fixed, resolving :)